scripts/gis-material-update/validate-deploy/modules/autoliikennemaarat.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from modules.config import Config
+from modules.common import *
+from modules.gis_validate_deploy import GisProcessor
+import geopandas as gpd
+
+class MakaAutoliikennemaarat(GisProcessor):
+    """Process traffic (car) volumes."""
+
+    def __init__(self, cfg: Config):
+        self._module = "maka_autoliikennemaarat"
+        self._buffers = cfg.buffer(self._module)
+        GisProcessor.__init__(self, cfg)
+
+    def get_temp_data(self, cfg):
+        self._tormays_files_temp = {}
+        self._filename = cfg.target_buffer_file(self._module)
+        for buffer in self._buffers:
+            buffer_data = gpd.read_file(self._filename.format(buffer))
+            buffer_data.rename_geometry('geom', inplace=True)
+            self._tormays_files_temp[buffer] = buffer_data
+
+    def validate_deploy(self):
+
+        # validate data amount: is it between given limits
+        buffers = self._buffers
+        self._validate_result = False
+
+        for buffer in buffers:
```

Comment:
This can use `self._buffers` directly. There's no need to assign it to the `buffers` variable.
```suggestion
        for buffer in self._buffers:
```

Reply:
changed